docs/quickstart.md

```diff
@@ -136,6 +143,55 @@ TODO(gfariasalves): Add instructions about installing a CNI or available flavour
 kubectl delete cluster ionos-quickstart
 ```
 
+## Cluster templates
+
+We provide various templates for creating clusters. Some of these templates provide you with a CNI already.
+
+For templates using `CNIs` you're required to create `ConfigMaps` to make `ClusterResourceSets` available.
+
+To enable `ClusterResourceSets` feature,
```

Comment:
This is enabled by default.
https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html?highlight=ClusterResource#experimental-feature-clusterresourceset-beta

Reply:
Good to know,

But this line won't have any affect whether to set it or not.